matminer/featurizers/composition/element.py

```diff
@@ -207,12 +209,24 @@ class BandCenter(BaseFeaturizer):
         - Band center
     """
 
-    magpie_data = MagpieData()
-    deml_data = DemlData()
+    def __init__(self, impute_nan=False):
+        self.impute_nan = impute_nan
+        if not self.impute_nan:
+            warnings.warn(
+                f"""{self.__class__.__name__}(impute_nan=False):
+                    In a future release, impute_nan will be set to True by default.
+                    This means that features that are missing or are NaNs for elements
+                    from the data source will be replaced by the average of that value
+                    over the available elements.
+                    This avoids NaNs after featurization that are often replaced by
+                    dataset-dependent averages."""
+            )
```

Comment:
Could we tidy this warning into a utils variable just so it is easier to change in the future?

```suggestion
            warnings.warn(
                f"{self.__class__.__name__}(impute_nan=False):\n" + IMPUTE_NAN_WARNING
            )
```

Reply:
Yep, much cleaner